Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -321,6 +329,11 @@ open class AVFoundationPlayback: Playback {
         trigger(.positionUpdate, userInfo: ["position": CMTimeGetSeconds(time)])
     }
 
+    open override func seekToLivePosition() {
+        play()
```

Comment:
Hey @nabilsafatli and @dezouzart , why do need to always call `play` here? This will always trigger events related with play. Would we check if it is paused to call play?

Reply:
@igoriols seeking to live should resume the playback. Otherwise, it would keep in DVR, only paused at the live position at the moment of the seek.

Reply:
Hey @igoriols ,
When the user ask to seek to live position we assume that the player needs to play again. Otherwise the player will come to a dvr state again.